Assets/QA/Input_Test/Created by @jia.txt

```diff
@@ -8,13 +8,25 @@ Q: Who made this?
    Please email me any comment/concern or poke me in Slack directly.
 
 Q: How to use this?
-   Go to folder: Input_Test -> Scenes and open the scene in there.
-   Since the new Input System does not support the UI element yet, I suggest use this with the old Input Manager.
-   The setting can be changed at: Player Settings -> Other Settings -> Active Input Handling
+   When using New Input System, use Ctrl+[num] to switch input method test, and Ctrl+I to see More Information.
+     Ctrl + 1: Keyboard and mouse
+     Ctrl + 2: Xbox controller
+     Ctrl + 3: General controller diagram
+     Ctrl + 4: Pen
+     Ctrl + 5: Touch
+	 Ctrl + I: More Information
+   When using Old Input Manager, use the Dropdown UI element to switch.
+   When using both, either/or would work.
+   *The setting can be changed at: Player Settings -> Other Settings -> Active Input Handling
+
+Q: How to setup the Old Input Manager?
+   You can download the InputManager.asset file from here: https://oc.unity3d.com/index.php/s/K5sMSRez4nJsMnT
+   And replace the one in the [project folder]->ProjectSettings folder.
+   WARNING: this could affect other tests in the same project. Be sure to make backup before replacing.
```

Comment:
I suggest putting the asset into the repo (probably in some toplevel folder; e.g. "QA/InputManager.asset" or something). Even better would be to not rely on manual procedures for these things but rather have menu items (e.g. a "QA" toplevel menu) that drive various QA automation tasks (such as building a player with the QA scenes without having to modify the global scene list).